follej/session04/trigram.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import random
+
+book_file = "./sherlock_small.txt"
+# book_file = "./sherlock_small.txt"
+
+
+word_pairs = {}
+
+
+def read_book():
+    with open(book_file, 'r') as f:
+        book_text = f.read().replace('\n', ' ')
+    book_text = book_text.lower()
+    # book_text = book_text.replace('.', '')
+    book_text = book_text.replace(',', '')
+    book_text = book_text.replace('"', '')
+    book_text = book_text.replace('!', '')
+    book_text = book_text.replace('?', '')
+    book_text = book_text.replace('  ', ' ')
+    book_text = book_text.replace('  ', ' ')
+    book_text = book_text.replace("'", '')
+    book_text = book_text.replace("(", '')
+    book_text = book_text.replace(")", '')
+    book_text = book_text.replace(";", '')
+    book_text = book_text.replace("-", ' ')
+    return book_text
```

Comment:
Can you think of a way to reduce this to two statements?  Think of a more "pythonic" approach.